com.kjxbyz.PicGuard.yml

```
@@ -0,0 +1,72 @@
+id: com.kjxbyz.PicGuard
+
+# https://docs.flatpak.org/en/latest/available-runtimes.html
+runtime: org.freedesktop.Platform
+runtime-version: '22.08'
```

```suggestion
runtime-version: '23.08'
```

22.08 will be EOL in a month